### DISCUSSION TITLE
脚注可否强制同页

### DISCUSSION THREAD

**Discussion by xx-liu** [tuna/thuthesis#774] [Q&A] [answered]
overleaf平台，7.2.4版本
用\footnote写脚注，该页有3条脚注，分别为2行、1行、3行，最后一条的最后一行跳到了下一页，请问可以让脚注强制同页吗？

**Comment by zepinglee** [comment]
LaTeX 的脚注比较复杂，需要给出复现的代码和截图，方便研究。

**Reply by xx-liu** [reply]
好像不太好复现，我猜是因为当页有图片的原因，调整了图的位置就好了。

**Reply by zepinglee** [reply, marked as answer]
4344d67 中改为严格禁止脚注跨页（#778），应该不会出现这个问题了。
